.github/workflows/docs.py

```diff
@@ -0,0 +1,36 @@
+"""Script to generate docs based on given input directory (first argument)
+and output directory (second argument)"""
+import os
+import subprocess
+import sys
+import glob
+import nbformat
+
+code_path = sys.argv[1]
+out_path = sys.argv[2]
+
+os.chdir(code_path)
```

Comment:
it could be cleaner and more readable to add the path to all paths below, so that the `chdir` would not be needed at all